org.adempiere.base.process/src/org/adempiere/process/InvoiceGenerateRMA.java

```diff
@@ -74,11 +74,14 @@ else if (name.equals("DocAction"))
                 log.log(Level.SEVERE, "Unknown Parameter: " + name);
         }
         
-        m_dateinvoiced = Env.getContextAsDate(getCtx(), Env.DATE);
-        if (m_dateinvoiced == null)
-        {
-            m_dateinvoiced = new Timestamp(System.currentTimeMillis());
+        if (m_dateinvoiced == null) {

```

Comment:
this change is strange, I don't see there's other changes that can make m_dateinvoiced != null at this point.